Modules/Sources/NotificationServiceExtensionCore/NotificationServiceExtensionConfiguration.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import BuildSettingsKit
+
+extension BuildSettings {
+    public var notificationServiceExtensionConfiguration: NotificationServiceExtensionConfiguration {
+        switch brand {
```

Comment:
We'll likely end up making more runtime decisions what to do based on what app is running. Should it be renamed?